Update documentation regarding manual plugin installation (#474)
jellyfin/jellyfin-docs: general/server/plugins/index.md

Many plugins are available in a repository hosted on our servers, which can be easily installed using the plugin catalog in the settings. At the moment many of these are still being updated frequently so the version number may not be accurate. There are several different categories that can indicate what kind of functionality the plugins may provide.

Once a plugin has been installed, removed, or upgraded, a restart of Jellyfin is required in order for the changes to take effect.

The *plugins folder* is located in different locations depending on your install:

> * `%UserProfile%\AppData\Local\jellyfin\plugins` for direct installs

Reviewer: this feels windows specific, we should specify each system. the linux path is buried in a paragraph

Author: Agreed, I'm not sure what the correct paths would be though.

Reviewer 2: The 2 paths there are 100% windows specific (IIRC I touched these Paths of this file some time ago due to some Reddit discussion I was part of).
Regardless, adding the Linux paths should be easy right?

* `%ProgramData%\Jellyfin\Server\plugins` for tray installs

> Plugin settings are stored in the `<direct or tray path>\plugins\configurations` folder, with the `.xml` file extension.

Reviewer 3: We should default to Linux style paths with forward slashes, and the rest of the docs use `{BRACKETS}` for variables if I recall correctly.

### Manual

> All plugins hosted on the repository can be built from source and manually added to your server as well. They just need to be placed in the plugin directory, which is something like `var/lib/jellyfin/plugins` on most Linux distributions. **Make sure to create a subdirectory for each plugin.** Once the server is restarted any additions should automatically show up in your list of installed plugins.

Reviewer: ```suggestion
All plugins hosted on the repository can be built from source and manually added to your server as well. They just need to be placed in the plugin directory, which is something like `var/lib/jellyfin/plugins` on most Linux distributions. **Make sure to create a subdirectory for each plugin.** Once the server is restarted, any additions should automatically show up in your list of installed plugins.
```

#### Troubleshooting plugins

> If you can't see the new plugin in the dashboard, then the first port of call should be the `meta.json` which Jellyfin automatically creates in the plugin folder.

Reviewer: ```suggestion
If you can't see the new plugin in the dashboard, then the first place to look should be the `meta.json` which Jellyfin automatically creates in the plugin folder.
```

If you cannot locate this file, then check the file and folder permissions to ensure that Jellyfin has access to the plugin folder.

The value in the `status` field of the meta.json file points to what is happening with the plugin.  

> * Restart - The plugin requires Jellyfin to be restarted.

Reviewer 3: I believe other lists like this use colons instead of dashes in our documentation.

* Active - The plugin is running as expected.
* Disabled - The plugin has been disabled, so won't be loaded.

> * NotSupported - The plugin does not meet the minimum requirements for your version of Jellyfin. Check is there is another version available, or contact the plugin developer.

Reviewer: ```suggestion
* NotSupported - The plugin does not meet the minimum requirements for your version of Jellyfin. Check in there if another version available, or contact the plugin developer.
```

> * Malfunction - The plugin caused an error during that last startup, so has been disabled. The error should be recorded in the system log.

Reviewer: ```suggestion
* Malfunction - The plugin caused an error during that last startup, so it has been disabled. The error should be recorded in the system log.
```

* Superceded - There is another version of the plugin installed. (Only one version of the plugin should be run at a time).

> * Deleted - The plugin is marked for deletion, and will be removed at the next restart.

Reviewer: can this be stopped if wanted?

Author: CC @BaronGreenback
